en/ros/ros2_comm.md

```diff
@@ -1,340 +1,211 @@
-# PX4-ROS 2 Bridge
+# ROS 2 User Guide (PX4-ROS 2 Bridge)
```

Comment:
@mrpollo This is pretty much done. Brutal update to rip out most of the fluff. This is now high level overview of what you actually need to do. 

- The overview contains all the background on how it works/what the packages are. All the other duplication removed.
- Build is short - step 1, 2, go
- I put sanity check as part of building. Might be worth moving out. 
- The manual build is at the end. NOt particularly important to users.
- What messages supported now just a link to the repo which shows the actual list px4_msgs. If we want to talk about generating this set, that would be a new topic - ie I don't care about Yaml.